test-data/jump.yaml

```diff
@@ -760,7 +760,10 @@ code:
   <exit>: |
     exit
 
-post: []
+post:
+  - r1.ctx_offset=0
+  - r1.svalue=[1, 2147418112]
+  - r1.type=ctx
```

Comment:
This seems correct to me, but I'm not 100% sure, and I don't know why this was not already the case. This PR is supposed to only be a refactoring, so I may have introduced a bug.
@Alan-Jowett what do you think?

Reply:
I'm merging the PR despite this question. If needed it will be fixed later on.

Reply:
Turns out I broke assume-assert. Will fix soon.